jet/kafka-connect/src/main/java/KafkaConnectSample.java

```diff
@@ -0,0 +1,88 @@
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.JetService;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.kafka.connect.KafkaConnectSources;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.StreamStage;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.neo4j.driver.AuthTokens;
+import org.neo4j.driver.Driver;
+import org.neo4j.driver.GraphDatabase;
+import org.neo4j.driver.Session;
+
+import java.net.URL;
+import java.util.Properties;
+
+
+/**
+ * Demonstrates how to use Neo4j as a source for Jet pipelines using Kafka Connect connector
+ * <p>
+ * Prerequisite: start a Neo4j container using the following command:
+ * <pre>
+ * docker run --rm -e NEO4J_AUTH=none --publish=7474:7474 --publish=7687:7687 neo4j:5.5.0
+ * </pre>
+ * <p>
+ * To generate a new items externally go to <a href="http://localhost:7474/browser/">Neo4j Browser</a>
+ * (no authentication required) and execute the following query
+ * <pre>
+ *     CREATE (:TestSource {name: 'my-name', value: 'my-value', timestamp: datetime().epochMillis})
+ * </pre>
+ */
+public class KafkaConnectSample {
+
+    //You can download the newer version from https://www.confluent.io/hub/neo4j/kafka-connect-neo4j/
```

Comment:
Is this using older version because we are still on Java 8?
Can you add this to the comment and create an issue to update this when we migrate to 11.

Reply:
Added https://github.com/hazelcast/hazelcast-code-samples/pull/569/commits/8ad7a11bea5bb8221ded3a3636901a77323491cd